.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "./vendor/gems/"
```

Comment:
I don't understand how these changes to bundle related to sunsetting the site.  Did you include them by mistake?